### PR TITLE
create-react-app issue #3663 - Conflict with jsx solved

### PR DESCRIPTION
**Pull request by LeoCR**
Hi everyone:

First of all, Happy new year
I found a conflict with the lasts versions of node, npm, react and babel with this package so I fix it with `.babelrc` file and added and removed some dependencies in the `package.json` file.

This is the issue that I had a few days ago: https://github.com/facebookincubator/create-react-app/issues/3663

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
 node_modules
 dist
 publish
+.DS_Store
+dest
```
Comment (nwwells) [inline .gitignore]: What's `dest`?

Comment (LeoCR) [reply .gitignore]: the gulpfile.js had it as destination folder from the begining I just add the .DS_Store so you can remove the ./dest/ if you wish . 
gulpfile

**Comment by antoinerousseau** [comment]
@LeoCR your diff is modifying most of the files in an unrelated way. Make sure you only relate to the said issue and we'll look into it. Let me know if you need help with git squashing & cleaning.

**Comment by LeoCR** [comment]
Hi Antoine Rousseau,you can reproduce the error that I had if you change the dependency into the package.json  with this repository https://github.com/LeoCR/react-transitions/blob/master/package.json 



Noted that I used my own React-Icons repository inside the DevDependencies:

> "devDependencies": {
>     "babel-cli": "^6.26.0",
>     "babel-preset-minify": "^0.2.0",
>     "babel-preset-react": "^6.24.1",
>     "react-icons": "git+https://github.com/LeoCR/react-icons.git",
>     "react-scripts": "^1.0.17"
> }

If you keep my actual DevDependencies all works fine when you run: `npm run start`

But if you like using https://github.com/gorangajic/react-icons as dependency instaed of my own solution probably you can see an error when you run `npm run start `

**Comment by antoinerousseau** [comment]
I understand you had a bug, I'm just saying this PR is not valid, look at the diff, many unrelated files have changes, including some unresolved conflicts
https://github.com/gorangajic/react-icons/pull/125/files

**Comment by LeoCR** [comment]
hi bro, that's true, so don't worry about it, maybe my proyect repo [https://github.com/LeoCR/react-transitions](https://github.com/LeoCR/react-transitions)  is so stranger .
I just want fix my issue for running the proyect repository bellow, but if you can reproduce the error probably you can see the same error that I had .Noted that I just overwrite the  font files .js with babel and webpack for remove these errors ,maybe I was wrong but works for me.

**Comment by johnunclesam** [comment]
@LeoCR any news on this create-react-app's side?

**Comment by antoinerousseau** [comment]
You still have commited conflicts in the changed files. Closing in favor of #141.

**Pull request closed without merging**